charts/ingress-nginx/templates/controller-configmap-cron.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    {{- include "ingress-nginx.labels" . | nindent 4 }}
+    app.kubernetes.io/component: controller
+    {{- with .Values.controller.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  name: {{ template "ingress-nginx.wallarmControllerCronConfig" . }}
+  namespace: {{ .Release.Namespace }}
+data:
+  crontab: |
+    # export-environment
+    0 */1 * * * timeout 10m /opt/wallarm/ruby/usr/share/wallarm-common/export-environment -l STDOUT
```

Comment:
@dmitry-j-mikhin Do we need to template cron expressions and timeouts here?

Reply:
@MrEcco why you want to merge this PR to ingress repo? I think you should create separate repo for sidecar and merge this PR to sidecar repo.

Reply:
We want to use the same images for new schema then it possible. I may reduce the complexity of implementation.

Reply:
@MrEcco resolved